roles/openshift_etcd_migrate/templates/validatemigration.py.j2

```diff
@@ -0,0 +1,91 @@
+from subprocess import PIPE, Popen
```

Comment:
I have to admit that generating code via template makes me a little uneasy. What about passing input into the code instead?